packages/valory/skills/registration_abci/tests/test_behaviours.py

```diff
@@ -276,8 +276,11 @@ def mock_get_tendermint_info(self, *addresses: str) -> None:
             info = json.dumps(DUMMY_VALIDATOR_CONFIG)
             response_kwargs = dict(info=info)
             self.mock_tendermint_request(request_kwargs, response_kwargs)
-        time.sleep(self.state.params.sleep_time)
-        self.behaviour.act_wrapper()
+        # give room to the behaviour to finish sleeping
+        # using the same sleep time here and in the behaviour
+        # can lead to problems when the sleep here is finished
+        # before the one in the behaviour
+        time.sleep(self.state.params.sleep_time * 2)
```

Comment:
Same problem as in the other test.

Reply:
The sleep time is tweaked in the `setup` and set to a very small value:

https://github.com/valory-xyz/open-autonomy/blob/5da80711d0c0c0cdfdca69ce7cd26aa5e917812f/packages/valory/skills/registration_abci/tests/test_behaviours.py#L163

Do you think that `*2` would suffice? It should though.